### ISSUE TITLE
NoSuchMethodError 1.19.2

### ISSUE THREAD

**Issue by Bloodlex** [LOOHP/InteractionVisualizer#93]
Hey,

I have a following error. Exception and details are below. Could you please take a look on that? 🙂 .

Version: git-Purpur-1858 (MC: 1.19.2)*
Interaction Visualizer: 1.18.2

```
16.12 14:30:27 [Server] WARN Task #274 for InteractionVisualizer v1.18.2.0 generated an exception
16.12 14:30:27 [Server] INFO java.lang.NoSuchMethodError: 'org.bukkit.craftbukkit.v1_19_R2.entity.CraftEntity net.minecraft.world.entity.Entity.getBukkitEntity()'
16.12 14:30:27 [Server] INFO at com.loohp.interactionvisualizer.nms.V1_19.lambda$getEntities$2(V1_19.java:225) ~[InteractionVisualizer-1.18.2.0.jar:?]
16.12 14:30:27 [Server] INFO at com.loohp.interactionvisualizer.objectholders.WrappedIterable$1.next(WrappedIterable.java:53) ~[InteractionVisualizer-1.18.2.0.jar:?]
16.12 14:30:27 [Server] INFO at com.loohp.interactionvisualizer.entities.ItemDisplay$1.run(ItemDisplay.java:162) ~[InteractionVisualizer-1.18.2.0.jar:?]
16.12 14:30:27 [Server] INFO at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[purpur-1.19.2.jar:git-Purpur-1858]
16.12 14:30:27 [Server] INFO at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[purpur-1.19.2.jar:git-Purpur-1858]
16.12 14:30:27 [Server] INFO at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1500) ~[purpur-1.19.2.jar:git-Purpur-1858]
16.12 14:30:27 [Server] INFO at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:486) ~[purpur-1.19.2.jar:git-Purpur-1858]
16.12 14:30:27 [Server] INFO at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1424) ~[purpur-1.19.2.jar:git-Purpur-1858]
16.12 14:30:27 [Server] INFO at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1194) ~[purpur-1.19.2.jar:git-Purpur-1858]
16.12 14:30:27 [Server] INFO at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:310) ~[purpur-1.19.2.jar:git-Purpur-1858]
16.12 14:30:27 [Server] INFO at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by LOOHP** [comment]
Fixed in 1.18.2.1

**Comment by Bloodlex** [comment]
> Fixed in 1.18.2.1

Okay, thanks! 🙂
